examples/e-commerce/package.json

```diff
@@ -10,9 +10,9 @@
   "dependencies": {
     "algoliasearch": "3.33.0",
     "classnames": "2.2.6",
-    "instantsearch.js": "3.5.3",
+    "instantsearch.js": "^3.6.0",
```

Comment:
We can pin this to 3.7.0 now (likely the last v3 version).

Reply:
I put the exact same value as `vue-instantsearch` to avoid any duplicate in the lockfile.